dali/pipeline/operators/sequence/seq_rearrange_test.cc

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <iostream>
+
+#include "dali/test/dali_operator_test.h"
+#include "dali/test/dali_operator_test_utils.h"
+#include "dali/pipeline/operators/sequence/seq_rearrange.h"
```

Comment:
generally, I'd prefer to name the file sequence_rearrange.h

Reply:
Done